src/main/kotlin/nl/info/zac/app/zaak/converter/RestZaakConverter.kt

```diff
@@ -216,4 +215,8 @@ class RestZaakConverter @Inject constructor(
             ?.forEach(gerelateerdeZaken::add)
         return gerelateerdeZaken
     }
+
+    private fun shouldSetOntvangstbevestigingNietVerstuurdIndication(zaak: Zaak, statustype: StatusType?) =
```

Comment:
Why the `should` in this function name? 'Should' is something to use in tests but not in real code? Code 'should' not do anything.